code/datums/spells/area_teleport.dm

```diff
@@ -78,7 +82,7 @@
 
 /obj/effect/proc_holder/spell/area_teleport/invocation(mob/user)
 	if(!invocation_area || !selected_area)
-		..()
+		return
 	else
```

Comment:
This else is now redundant, you can remove it and remove an indent from everything it had within